engine/src/cmd/basecomputer.cpp

```diff
@@ -5250,53 +5250,53 @@ void showUnitStats( Unit *playerUnit, string &text, int subunitlevel, int mode,
             break;
         }
     }
-    if (!mode || MODIFIES(replacement_mode, playerUnit, blankUnit, armor.facets[2].health.health)) {
+    if (!mode || MODIFIES(replacement_mode, playerUnit, blankUnit, layers[1].facets[2].health.health)) {
```

Comment:
`armor` was more descriptive; though I like the array. The descriptiveness can be continued if you use a constant or enum for the index instead. Same for the shields below.

Reply:
Fixed by adding shorthand pointers hull, armor and shield to DamageableLayer.